ext/brotli/extconf.rb

```diff
@@ -7,12 +7,18 @@
 create_makefile('brotli/brotli')
 
 def acopy(dir)
+  # __dir__ is only supported >= ruby 2.0, use __FILE__ approach
+  # Gem should support wide range of ruby versions
+  def _dir
+    File.dirname(__FILE__)
+  end
```

Comment:
I think this func does not need. just write `File.dirname(__FILE__)` as like spec file.